lineman/app/components/invitation_form/add_members_modal/add_members_modal.coffee

```diff
@@ -0,0 +1,35 @@
+angular.module('loomioApp').factory 'AddMembersModal', ->
+  templateUrl: 'generated/components/invitation_form/add_members_modal/add_members_modal.html'
+  controller: ($scope, Records, LoadingService, group, AppConfig, FlashService) ->
+    $scope.isDisabled = false
+    $scope.group = group
+    $scope.loading = true
+    $scope.selectedIds = []
+
+    $scope.load = ->
+      Records.memberships.fetchByGroup(group.parent().key, {per: 500})
+
+    $scope.members = ->
+      _.filter group.parent().members(), (user)->
+        !user.isMemberOf(group)
+
+    $scope.canAddMembers = ->
+      $scope.members().length > 0
+
+    LoadingService.applyLoadingFunction($scope, 'load')
+    $scope.load()
+
+    $scope.submit = ->
```

Comment:
Couldn't get FormService to fit here?

I believe there's a successFn option which is called after success which passes the data through... maybe the addUsersToSubgroup submitFn was tricky?